tests/unit/htdocs/modules/protector/Protector364RcFixesTest.php

```
@@ -0,0 +1,526 @@
+                'AUTH_TYPE',
+                'QUERY_STRING',
+                'REQUEST_URI',
+                'PATH_INFO',
+                'PATH_TRANSLATED',
+                'ORIG_PATH_INFO',
+                'PHP_SELF',
+                'SCRIPT_NAME',
+                'SERVER_NAME',
+                'REQUEST_METHOD',
+                'SERVER_PROTOCOL',
+            ] as $attackerControlledKey
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $attackerControlledKey. Keep variable name length under 20. (undefined)

(LongVariable)